presto-hive/src/main/java/com/facebook/presto/hive/InternalHiveSplit.java

```diff
@@ -86,7 +88,7 @@ public InternalHiveSplit(
         requireNonNull(schema, "schema is null");
         requireNonNull(partitionKeys, "partitionKeys is null");
         requireNonNull(blocks, "blocks is null");
-        requireNonNull(bucketNumber, "bucketNumber is null");
+        requireNonNull(readBucketNumber, "readBucketNumber is null");
```

Comment:
nit: add null check for `partitionBucketNumber` as well

Reply:
Will do